_integrations/index.md

```diff
@@ -9,60 +9,116 @@ redirect_from:
 
-Using the OpenSearch Dashboards interface, you can connect your data, applications, and processes so that you can centrally manage the services you use. All integrations are available in a single view, and OpenSearch Dashboards guides you there from the home page and main menu. 
+These pipelines use the OTel schema (or a simple schema) to index signal documents into the correct index representing the observed resource signals. See [Naming Convention](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/schema/observability/Naming-convention.md) for index naming conventions. 
 
-Learn how to do the following using the OpenSearch Dashboards interface:
+### Ingestion structure
 
-- Access integrations
-- View integrations
-- Add integrations 
+Each integration contains the following metadata and assets:
 
-The following image gives you a snapshot of the Integrations interface: 
+* Name and description
+* Source URL and license
+* Schema specification, for example, mapping or component mapping
+* Sample data for testing the feature
+* Assets such as dashboards, index patterns, queries, or alerts
 
-![Get started with Integrations demo]({{site.url}}{{site.baseurl}}/images/integrations/nginx-integration.gif)
+---
+
+## Using Integrations
 
-### Access integrations
+**Integrations** can be installed directly from the [default catalog](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/integrations/Release.md) shipped with every OpenSearch release. Additionally, **Integrations** can be manually loaded and updated using the **Dashboards Management** console.
 
-To access integrations, open OpenSearch Dashboards and select **Integrations** from the **Management** menu. The interface displays installed and available integrations.
+### Installing integrations 
+
+If no integration is installed, then you are prompted to install one. Go to **Integrations** > **Available** to view the supported integrations available from the default catalog. This feature is shown in the following image.
+ 
+![]({{site.url}}{{site.baseurl}}/images/integrations/empty-installed-integrations.png)
 
-## View integrations
+To add an integration, select the desired prepackaged asset. **Integrations** offers the following options: **Add** or **Try it**. You can choose **Add** to add or configure a new data integration by selecting a prepackaged integration asset. You can choose **Try it** to to test or explore the integration before fully adding it. This feature is shown in the following image.
 
-To view an integration, view the dashboard associated with the integration. If the integration doesn't have an associated dashboard, select the desired integration listed under the **Installed** window. View the integration details, such as assets and fields. 
+![integrations-observability-catalog.png](/images/integrations/integrations-observability-catalog.png)
```

Comment:
@YANG-DB This image differs from what I see in the nightly playground. Please let me know if I should update this image. 

<img width="1606" alt="Screenshot 2024-06-18 at 6 13 45 PM" src="https://github.com/opensearch-project/documentation-website/assets/105296784/9f077455-854c-4f68-9b97-1d2816b4fcbd">

Reply:
keep the image - its up to date